### PR TITLE
Implements the new EIP1559 UI components

### PR DESCRIPTION
**Pull request by darkwing**
## Customize Gas Modal:

https://user-images.githubusercontent.com/46655/123339775-cd91a780-d510-11eb-800b-7a37748ec866.mp4

## Speed Up Transaction:

https://user-images.githubusercontent.com/46655/123339787-d6827900-d510-11eb-9482-0a0ac64cdccc.mp4

### CODE REVIEW

File: ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js [hunk 1]
```diff
@@ -24,52 +28,66 @@ export default function AdvancedGasControls() {
...
-              variant={TYPOGRAPHY.H8}
-            ></Typography>
-          </>
-        }
-      />
+      {process.env.SHOW_EIP_1559_UI ? (
+        <>
+          <AdvancedGasControlsRow
+            titleText={t('maxPriorityFee')}
+            tooltipText=""
```
Comment (brad-decker) [inline ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js:35]: [Q] Are we just waiting for tooltip copy here?

File: ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js [hunk 2]
```diff
@@ -24,52 +28,66 @@ export default function AdvancedGasControls() {
...
+              <>
+                <Typography
+                  tag="span"
+                  color={COLORS.UI4}
+                  variant={TYPOGRAPHY.H8}
+                  fontWeight={FONT_WEIGHT.BOLD}
+                >
+                  {t('gasFeeEstimate')}:
+                </Typography>{' '}
+                <Typography
```
Comment (brad-decker) [inline ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js:48]: [Q] Is this empty component a placeholder for a future string?

Comment (brad-decker) [reply ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js:48]: [A] I see now, this is where the estimate goes. Ignore me.

**Review by brad-decker — APPROVED** [review]
ç'est magnifique

File: ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js (hunk 1 above)
Comment (darkwing) [reply ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js:35]: Yes -- I didn't see copy for that in the Figma file.

**Comment by brad-decker** [comment]
Checked this out and applied the jest test fixes and everything is still green so 👍🏻

**Review by ryanml — APPROVED** [review]
bella!

**Pull request merged**